docs/installingNative.md

```diff
@@ -1,19 +1,24 @@
 # Running natively
 
-_WARNING:_ The recommended setup for PrairieLearn development is [within Docker](installingLocal.md). The setup described on this page is not recommended or supported.
-
 This page describes the procedure to install and run PrairieLearn without any use of Docker. This means that PrairieLearn is running fully natively on the local OS.
 
 - Install the prerequisites:
 
-  - [Node.js](http://nodejs.org/)
-  - [Yarn](https://classic.yarnpkg.com/lang/en/docs/install/)
-  - [PostgreSQL 15](https://www.postgresql.org)
+  - [Git](https://git-scm.com)
+  - [Node.js](http://nodejs.org)
+  - [Yarn](https://classic.yarnpkg.com)
   - [Python 3.10](https://www.python.org)
-  - [Graphviz](https://graphviz.org/download)
-  - command-line git or [GitHub Desktop](https://desktop.github.com)
```

Comment:
We mention GitHub Desktop but never mention what to actually do with it, so I opted to remove it in favor of recommending just the standard Git binary. If someone is doing a native install, they can almost definitely run a copy/pasted `git clone` command.